common/personalization.js

```diff
@@ -101,6 +108,13 @@ function sendDisplayEvent(aepEdgeClient, req, propositions, cookieEntries) {
             },
           },
         },
+        data : {
```

Comment:
`in-app-response-format` is only needed for the requests to get personalization.  not needed for display and interact events.